cmd/api.go

```diff
@@ -0,0 +1,92 @@
+package cmd
+
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	"github.com/kelseyhightower/envconfig"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/hellofresh/kangal/pkg/core/observability"
+	kube "github.com/hellofresh/kangal/pkg/kubernetes"
+	loadTestV1 "github.com/hellofresh/kangal/pkg/kubernetes/generated/clientset/versioned/typed/loadtest/v1"
+	"github.com/hellofresh/kangal/pkg/proxy"
+	"github.com/hellofresh/kangal/pkg/report"
+)
+
+type apiCmdOpts struct {
+	kubeConfig      string
+	masterURL       string
+	maxLoadTestsRun int
+}
+
+// NewAPICmd creates a new api command
+func NewAPICmd(ctx context.Context) *cobra.Command {
```

Comment:
Why add a new command for this and not add it to the current proxy as a flag that can be enabled?

Reply:
Since it is experimental it will either fail, so with the new command it would be esier to clean up, or succeed and then old proxy would be easier to remove =)

Reply:
Mhh not sure I agree with this but let's give it a shot.